integration/integration.test.js

```diff
@@ -0,0 +1,77 @@
+const { execSync } = require("child_process");
+const path = require("path");
+const fs = require("fs");
+
+test("Integration test", () => {
+  const start = process.hrtime();
+  const output = execSync(
```

Comment:
Why not putting the "setup"-phase in a `beforeAll` and then do the expections in `it`s?

Reply:
That would be a bit easier to read. Will fix tonight